direct.mk

```diff
@@ -79,12 +80,12 @@ build: ## build the go packages
 .PHONY: test
 test: ## run tests, except integration tests
 	@echo "🐳 $@"
-	@go test -parallel 8 ${RACE} -tags "${DOCKER_BUILDTAGS}" $(filter-out ${INTEGRATION_PACKAGE},${PACKAGES})
+	@go test ${GO_TESTFLAGS} -tags "${DOCKER_BUILDTAGS}" $(filter-out ${INTEGRATION_PACKAGE},${PACKAGES})
```

Comment:
Did you remove the `-parallel 8` on purpose here?

Reply:
My bad :persevere: